pkgs/by-name/tu/tusc-sh/package.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, writeShellApplication
+, curl
+, coreutils
+, jq
+}:
+
+let
+  tusc = stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "tusc-sh";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "adhocore";
+    repo = "tusc.sh";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-h2AZa/vYNNclP6eb6gZymin53YiST/BBHdQ7loXhDW4=";
+  };
+
+  dontConfigure = true;
+  dontBuild = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    mv tusc.sh tusc
```

Comment:
suggestion: now that you're using `writeShellApplication`, there's no need to rename `tusc.sh` to `tusc` (sorry this didn't occur to me before).

Reply:
Thank you for the tip! Just pushed that change.